src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/StringModule.fs

```diff
@@ -41,8 +41,8 @@ type StringModule() =
         let e6 = String.concat "foo" ["bar"]
         Assert.AreEqual("bar", e6)
         
-        let e7 = String.concat "foo" ["bav";"baz"]
-        Assert.AreEqual("bavfoobaz", e7)
+        let e7 = String.concat "foo" ["bar";"baz"]
+        Assert.AreEqual("barfoobaz", e7)
```

Comment:
I understand that, but this is a false positive. Changing code because a false positive of a tool is [insert curse-word] ;-)